packages/reaction-core/server/methods/cart.js

```diff
@@ -706,10 +848,13 @@ Meteor.methods({
    * and adds "paymentSubmitted" to cart workflow
    * Note: this method also has a client stub, that forwards to cartCompleted
    * @param {Object} paymentMethod - paymentMethod object
+   * @param {String} sessionId - current client session id. We don't use it
+   * directly within this method, just throw down though hooks
    * @return {String} returns update result
    */
-  "cart/submitPayment": function (paymentMethod) {
+  "cart/submitPayment": function (paymentMethod, sessionId) {
```

Comment:
a breaking change could be prevented here with a:

- adapt to using an object on these external method calls so that it's not a breaking change in the future
- use something like `sessionId = sessionIdArg || Session.get('sessionId');` as `Session` is global.
-